evap/staff/tests/test_live.py

```diff
@@ -71,3 +84,28 @@ def test_submit_changes_form_data(self):
         self.assertEqual(contribution1.order, 0)
         self.assertEqual(contribution1.role, Contribution.Role.EDITOR)
         self.assertEqual(contribution1.textanswer_visibility, Contribution.TextAnswerVisibility.GENERAL_TEXTANSWERS)
+
+    def test_staff_semester_view_columns_not_searchable(self):
+        """Regression test for #2461"""
+
+        semester = baker.make(Semester)
+        course = baker.make(Course, semester=semester, name_en="course name")
+        baker.make(Evaluation, course=course, name_en="evaluation name")
+
+        with self.enter_staff_mode():
+            self.selenium.get(self.live_server_url + reverse("staff:semester_view", args=[semester.pk]))
+
+        search_input = self.wait.until(
+            visibility_of_element_located((By.CSS_SELECTOR, "input[type='search'][name='search-evaluation']"))
+        )
+        search_input.clear()
+        search_input.send_keys("course name")
+
+        self.wait.until(visibility_of_element_located((By.ID, "evaluation-table")))
+        self.wait.until(visibility_of_element_located((By.XPATH, "//td//a[contains(text(),'course name')]")))
```

Comment:
The test is really cool!

Can we maybe add an assertion to this first block that one of the visible table cells contains the word "exam", so that if we change the wording in the future, the test fails (instead of just always passing because the invisibility assertion below trivially always holds)?